lua/entities/gmod_tardis_interior/modules/cl_tips.lua

```diff
@@ -34,6 +55,9 @@ function ENT:InitializeTips(style_name)
 		for setting,value in pairs(interior_tip) do
 			tip[setting]=value
 		end
+		if tip.control then
+			tip.text = tip_control_texts[tip.control] or ""
```

Comment:
the fallback should be clear it doesn't exist, should be like "unknown control: ".. tip.control

Reply:
Added an error